docs/skyrim_platform/texts.md

```diff
@@ -8,8 +8,9 @@ You can convert a `*.ttf` to this format using MakeSpriteFont utility that is
 provided by DirectXTK. See [Compiling Font](#compiling-font) for details.
 
 ## Example
+
 ```typescript
-skyrimPlatform.createText(0, 0, "Hello", [1,1,0,1]); // 0,0 is top left. Non-ASCII character are not yet supported.
+skyrimPlatform.createText(0, 0, "Hello World!", [1, 1, 1, 1], "Data/Platform/Fonts/Tavern.spritefont"),; // 0,0 is top left. Non-ASCII character are not yet supported.
```

Comment:
Probably it makes no sense to pass a full path to the font here. Moreover, I think that just the name is enough. So the path would be assembled on the last step as `'Data/Platform/Fonts' + fontName + '.spritefont''`

Reply:
changed